addons/project/data/mail_template_data.xml

```diff
@@ -65,17 +65,17 @@
                     <table style="width:100%;text-align:center;margin-top:2rem;">
                         <tr>
                             <td>
-                                <a t-attf-href="/rate/{{ access_token }}/5">
+                                <a t-attf-href="/rate/{{ access_token }}/5" t-attf-class="{{ 'pe-none' if object._rating_get_operator().id else '' }}">
```

Comment:
```suggestion
                                <a t-attf-href="/rate/{{ access_token }}/5" t-att-class="{'pe-none': object._rating_get_operator().id}">
```
I think this is the simplest way to put it

Reply:
@auon-odoo,  tried above suggeston user can still able to click on emoji as  'pe-none'  class not applied so instead of that we can add condition like t-att-class="object._rating_get_operator().id and 'pe-none'".